gapic/src/main/com/google/gapid/views/ShaderView.java

```diff
@@ -153,16 +155,24 @@ public ShaderView(Composite parent, Models models, Widgets widgets) {
       }
     });
 
-    pushButton = Widgets.createButton(loading.getContents(), "Push Changes", e -> {
-      if (shaderResource != null && shaderMessage != null) {
-        models.analytics.postInteraction(View.ShaderView, ClientAction.Edit);
-        models.resources.updateResource(shaderResource, API.ResourceData.newBuilder()
-            .setShader(shaderMessage.toBuilder().setSource(spirvViewer.getDocument().get()))
-            .build());
-      }
-    });
-    pushButton.setLayoutData(new GridData(SWT.RIGHT, SWT.BOTTOM, false, false));
-    pushButton.setEnabled(false);
+    if (!Experimental.enableAll.get()) {
```

Comment:
Let's have a dedicated flag for this, e.g. `enableUnstableFeatures`, rather than relying on `enableAll`.